lib/compat/wordpress-6.3/class-gutenberg-rest-templates-controller-6-3.php

```diff
@@ -80,6 +82,14 @@ public function prepare_item_for_response( $item, $request ) { // phpcs:ignore V
 
 		$response = parent::prepare_item_for_response( $item, $request );
 
+		// Adds modified date to the response.
+		if ( rest_is_field_included( 'modified', $fields ) && ! empty( $response->data['wp_id'] ) ) {
```

Comment:
It seems like a good opportunity to add a rest prepare filter here, like we have for most(if not all) other controllers.
`return apply_filters( "rest_prepare_$item->type", $response, $item, $request );`

This way we would add the new field in the response with something like:
```
add_filter( 'rest_prepare_wp_template', 'filter_wp_template_response', 10, 2 );
add_filter( 'rest_prepare_wp_template_part', 'filter_wp_template_response', 10, 2 );
```

containing the logic for modified date. --cc @TimothyBJacobs

Reply:
This should all be doable with `register_rest_field` though.

Reply:
Oh, I completely forgot about that filter 🤦 

Thanks for nudging me in the right direction, @ntsekouras and @TimothyBJacobs . 

I've achieved the same result by returning the modified date in `get_callback` in   the `register_rest_field` hook.

```php
				'get_callback' => function( $object ) {
					if ( ! empty( $object['wp_id'] ) ) {
						$post = get_post( $object['wp_id'] );
						if ( $post ) {
							return $post->post_modified;
						}
					}
					return null;
				},
```

> This should all be doable with register_rest_field though.

Is that what you were recommending @TimothyBJacobs ?

Reply:
Yep! But it needs to be formatted correctly. See https://developer.wordpress.org/reference/functions/mysql_to_rfc3339/